Resources/lu-LaboratoryTestResult-DiagnosticReport.xml

```diff
@@ -0,0 +1,225 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="lu-LaboratoryTestResult-DiagnosticReport" />
+  <url value="http://nictiz.nl/fhir/StructureDefinition/lu-LaboratoryTestResult-DiagnosticReport" />
+  <name value="LuLaboratoryTestResultDiagnosticReport" />
+  <title value="lu LaboratoryTestResult DiagnosticReport" />
+  <status value="draft" />
+  <publisher value="Nictiz" />
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="url" />
+      <value value="https://www.nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="A Diagnostic Report for Laboratory exchange based on the Dutch Health and Care Information models (Dutch: Zorginformatiebouwsteen or ZIB)  version 4.6, release 2020." />
+  <purpose value="# Purpose of HCIM\n\nLaboratory tests are done for the purpose of diagnosing and preventing disease and follow-up on the effects of treatment." />

```

Comment:
I would expect an explanation for the existence of this profile (so not what it's for, but why it's needed).